### DISCUSSION TITLE
3DTileset 组件背景穿透问题

### DISCUSSION THREAD

**Discussion by 312837310** [zouyaoji/vue-cesium#212] [General]
Hello:

使用了VcPrimitiveTileset组件，加载3DTileset模型，然后影像使用高德卫星地图（其他地图应该一样），在缩放地图或调整视角下，会出现模型与地图穿透问题，这个有什么解决方案吗？比如通过某些参数控制

下图是实际开发的模型效果：
![image](https://user-images.githubusercontent.com/12478715/172978735-015b26f1-1b94-4b37-830d-5a8365224bff.png)

下图是官方的示例效果：
![image](https://user-images.githubusercontent.com/12478715/172978768-c98cc1fd-24be-4bbc-b862-512a3574f8af.png)

下图是第三方的demo：
![image](https://user-images.githubusercontent.com/12478715/172979287-b75f6885-d62a-49ff-b440-e392b874205e.png)

非常感谢

**Comment by zouyaoji** [comment]
看起来正常的那种，应该是没有开启深度检测吧。

![image](https://user-images.githubusercontent.com/24430116/173014563-4d92971a-d2a4-446a-813e-efdae9fd4791.png)

**Reply by 312837310** [reply]
首先感谢解答，我按照你的方案修改了还是存在这个问题，主要还是地球缩放和高度调整，模型会透出地图背景色；

这是vue-cesium中的demo效果，还是会出现那个情况，使用的是我自己的3DTiles三维模型
![image](https://user-images.githubusercontent.com/12478715/173178476-d84b59e1-268b-401b-8c6f-cede8aa2ebce.png)

这是缩放比较小的正常的效果，期望不管缩放多少，都是这种效果，模型层级最高，永远在上面
![image](https://user-images.githubusercontent.com/12478715/173178487-0be93632-b863-4dc2-b1c5-28151533670f.png)

非常感谢！

**Reply by zouyaoji** [reply]
没招了，vue-cesium是没有做特殊处理的，你可以拿原生的试试，估计也是这样。
